docs/user-guide/kubectl-overview.md

```diff
@@ -266,7 +266,28 @@ $ kubectl logs <pod-name>
 $ kubectl logs -f <pod-name>
 ```
 
+## Using kubectl with multiple remote environments
+
+When managing multiple Kubernetes clusters, it is very useful to be able to run kubectl on your local machine, and connect to different remote Kubernetes installations. First of all, [download and install](/docs/getting-started-guides/binary_release/) `kubectl` on your machine.
+You can now connect to a remote Kubernetes API server by providing the `--server` argument.
+
+```shell
+kubectl get pods --server=k8s.api.server:8080
+```
+
```

Comment:
The 8080 port is usually the insecure access channel, and in most cases is bound only to loopback.  Ideally we should add something about how to setup the credentials to talk to the secure port.  For now maybe just changing this to 127.0.0.1:8080 is sufficient.